Various improvements to Enforce and Override page (#1944)
hashicorp/terraform-website: content/source/docs/cloud/sentinel/enforce.html.md

> **Hands-on:** Try the [Enforce Policy with Sentinel](https://learn.hashicorp.com/collections/terraform/policy?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn.

> Policies are defined within policy sets which can be applied to selected or to all workspaces in an organization. Once a policy set is added to an organization, all of its policies are enforced on all runs in those workspaces to which the policy set is applied.

Reviewer: ```suggestion
You can define individual policies within larger policy sets and apply those policy sets to all or a subset of workspaces in an organization. Terraform Cloud then enforces all of those policies on every workspace run.  
```

> The policy checks will occur after a plan is successfully executed in the run. However, if cost estimates are enabled on the workspace, then the policy checks occur after the cost estimates are executed. This allows the policies to restrict costs based on the data in the cost estimates. If the plan fails, the policy checks will not be performed. The policy checks use data provided to Sentinel from the plan, state, configuration, workspace, and run to verify the rules in each of the policies.

Reviewer: ```suggestion
Policy checks occur after a plan and any enabled cost estimates are successfully executed in the run. This allows policies to restrict costs based on the data in the cost estimates. If the plan fails, Sentinel does not perform policy checks. The policy checks use data from the plan, state, configuration, workspace, and run to verify and enforce the rules in each policy.
```

> Enforcement level details can be found in the [Managing Policies](./manage-policies.html) documentation.

Reviewer: ```suggestion
Refer to the [Managing Policies](./manage-policies.html) documentation for more detail about enforcement.
```

> All `hard-mandatory` must pass in order for the run to continue to the "Confirm & Apply" state. All `soft-mandatory` policies must pass or be overridden for the run to contineu to the "Confirm & Apply" state.

Reviewer 2: ```suggestion
All `hard-mandatory` must pass in order for the run to continue to the "Confirm & Apply" state. All `soft-mandatory` policies must pass or be overridden for the run to continue to the "Confirm & Apply" state.
```

> If any `soft-mandatory` policies fail and no `hard-mandatory` policies fail, users with permission to override policies will be presented with an "Override & Continue" button in the run in the Terraform Cloud workspace. They have the ability to override the failed `soft-mandatory` policy checks and continue the execution of the run. This will not have any impact on future runs. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html)) Those same users can also override `soft-mandatory` policies when running the `terrform apply` command with the Terraform CLI by typing "override" when promoted to override failed `soft-mandatory` policies for the run.

Reviewer: Our new style guide tells us to use bold for interface elements instead of quotations. I know this is a change, so we're slowly migrating content from using quotations to using bold :-) 

```suggestion
If any `soft-mandatory` policies fail and no `hard-mandatory` policies fail, users with [permission to override policies](/docs/cloud/users-teams-organizations/permissions.html#manage-policy-overrides) will be presented with an **Override & Continue** button in the run in the Terraform Cloud workspace. This allows them to override the failed `soft-mandatory` policy checks and continue the execution of the run. This will not have any impact on future runs. 

These users can also override `soft-mandatory` policies by running the `terraform apply` command and then entering "override" when prompted to override failed `soft-mandatory` policies for the run.
```

[permissions-citation]: #intentionally-unused---keep-for-maintainers

> If an `advisory` policy check fails, it will show the warning state in the run; however, the execution of the run will continue to the "Confirm & Apply" state. No user action is required to override or continue the run execution.

Reviewer: ```suggestion
If an `advisory` policy check fails, it will show the warning state in the run, and the execution of the run will continue to the "Confirm & Apply" state. No user action is required to override or continue the run execution.
```
